apps/website/docs/react/api/make-static-styles.md

```diff
@@ -100,3 +100,44 @@ function App() {
   return <div />;
 }
 ```
+
+:::note
+
+`makeStaticStyles` supports the use of [shorthands](/react/api/shorthands) and does not have the same limitation as `makeStyles()`.
+
+:::
+
+## Usage with `makeStyles`
+
+```tsx
+import { makeStyles, makeStaticStyles, mergeClasses, shorthands } from "@griffel/react";
+
+const useStaticStyles = makeStaticStyles({
+  body: { color: "red", padding: "5px" }
+});
+
+const useClasses = makeStyles({
+  primary: { color: "blue" },
+  circular: {
+    ...shorthands.padding("500px"),
+    ...shorthands.borderRadius("0px")
+  }
+});
+
+export default function App(props) {
+  useStaticStyles();
+  const classes = useClasses();
+
+  return (
+    <>
+      <h1>Hello World!</h1>
+      <button
+        className={mergeClasses(
+          props.primary && classes.primary,
+          props.circular && classes.circular
+        )}
+      />
+    </>
+  );
+}
+```
```

Comment:
````suggestion
```tsx
import { makeStyles, makeStaticStyles, shorthands } from '@griffel/react';

const useStaticStyles = makeStaticStyles({
  body: {
    color: 'red',
    padding: '5px',
  },
});

const useClasses = makeStyles({
  primaryText: {
    color: 'blue',
    ...shorthands.padding('5px'),
  },
});

export default function App(props) {
  useStaticStyles();
  const classes = useClasses();

  return <p className={props.primaryText}>Hello world</p>;
}
```
````

Let's simplify the example to not include `mergeClasses()` as it's not related to the problem :)

Reply:
Okay. 👍